.travis.yml

```diff
@@ -5,6 +5,10 @@ cache:
   directories:
   - node_modules
   - bower_components
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
```

Comment:
test-headless should be doing this for you already https://github.com/openshift/origin-web-common/blob/master/hack/test-headless.sh#L34

and if you look at our travis.yml in origin-web-console this wasn't necessary